src/Components/App/App.js

```diff
@@ -1,42 +1,51 @@
 import React, { Component } from 'react';
-import movieData from '../../movieData'
+// import movieData from '../../movieData'
 import Header from '../Header/Header';
 import Movies from '../Movies/Movies';
 import Details from '../Details/Details'
+import { fetchAllMovies } from '../../apiCalls'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      allMovies: movieData.movies,
+      allMovies: [],
       isMovieSelected: false,
       selectedMovieID: null,
+      isLoading: true,
+      error: ""
     }
   }
 
-  toggleSeletion = (movieID) => {
+  toggleSelection = (movieID) => {
     this.setState({
       isMovieSelected: !this.state.isMovieSelected, 
       selectedMovieID: movieID,
      })
   }
 
+  componentDidMount = () => {
+    fetchAllMovies()
+      .then(movies => this.setState({ allMovies: movies.movies, isLoading: false }))
+      .catch(error => this.setState({ error: "These taters got too rowdy - check back later!"}))
+  }
+
```

Comment:
Cool to see this incorporating our fetch calls with React now!
Still curious if we may find ways to reflect more specific errors in a message to the user, but I do love this general one we have in for now!